meteor-app/server/api/v1/accounts/methods.ts

```diff
@@ -91,10 +99,10 @@ Meteor.methods({
 	},
 
 	Fetch_Viewers_For_Account: function () {
-		if (this.userId && Roles.userIsInRole(this.userId, ['Editor'])) {
-			let accountId = Meteor.users.findOne({ _id: this.userId })?.profile.accountId
-			return Meteor.users.find({ 'profile.accountId': accountId }).fetch()
-		}
+		enforceRole(this.userId, 'Editor')
+		let accountId = Meteor.users.findOne({ _id: this.userId })?.profile.accountId
```

Comment:
We should add a conditional to gaurd against if accountId is falsy. 

if no account Id, return empty array.

If no users with matching account Id, return empty array